newm-server/src/main/kotlin/io/newm/server/config/repo/ConfigRepository.kt

```diff
@@ -77,5 +77,7 @@ interface ConfigRepository {
         const val CONFIG_KEY_CLIENT_CONFIG_STUDIO = "clientConfig.studio"
         const val CONFIG_KEY_CLIENT_CONFIG_MARKETPLACE = "clientConfig.marketplace"
         const val CONFIG_KEY_CLIENT_CONFIG_MOBILE = "clientConfig.mobile"
+        const val CONFIG_KEY_RELEASED_EMAIL_NOTIFICATION_TITLE = "released.email.notification.title"
+        const val CONFIG_KEY_RELEASED_EMAIL_NOTIFICATION_TITLE_VALUE = "Subj: [IMPORTANT] Your song has been released!\n Preview: Now available on all major streaming platforms"
     }
```

Comment:
For consistency, follow the same patter we do with the other email notifications. At application.conf you will need to add a `released {...}` block under `mintingNotifications {...}`, similar to the other ones we have there for started, succeeded, ect. I'll send you more details via Slack.